### PR TITLE
Cuban Pete will no longer tolerate cheating at his arcade game.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

Your Gamer skill no longer gives you bonus damage during the fight against Cuban Pete.

## Why It's Good For The Game

Beating Cuban Pete gives you a max cap explosive. You need to earn these now.

## Changelog
:cl:
balance: Cuban Pete will no longer entertain Gamers :tm: doing more damage to him.
/:cl:

### CODE REVIEW

**Comment by nullAdept** [comment]
Is it even possible to win this now?

**Comment by necromanceranne** [comment]
It's not cheating, it's getting good.

**Comment by Winterous** [comment]
Two things.
1. This is not cheating.
2. This isn't going to make it harder, as your finishing blow is how you win, not raw damage.
3. There is an ACTUAL way of cheating that this doesn't affect.
4. What are you, drunk?

**Comment by Winterous** [comment]
Also even being ABLE to beat him takes a significant time investment, you realistically need at least master skill to have a chance, and that takes a while to build up; it's not for the damage, it's for the HEALTH.

Also also, making a feat which, let's be honest comes down to getting lucky, less likely to achieve, is not not forcing the player to "earn" the reward. There's no increased skill cap, it's just an increased luck requirement.

And again, this change doesn't even make it harder to win. Maybe very slightly, because the finishing blow should usually kill him.

**Comment by CRiPLe1000** [comment]
Hello, im the guy that bombed with Arcade Battle maxcaps back in March/April and was the reason (at least partially i hope) why Arcade Battle was reworked. I also bombed few times with reworked Arcade Battle and i will say one thing - this PR will change nothing and wont solve nothing. 

Right now gaming skill gives you extra health and damage which is fine as you need extra health to even survive. Now for the interesting bit - you dont win against Cuban Pete by dealing damage, you win by surviving long enough and getting all his traits disabled by going through combos (check original PR about rework for the way how to do that). Damage is not important at all in that regard which is why this change is useless or the impact will be limited at best.

Also you might think that removing Cuban Pete is another option so i will ask you a question: how many bombs are you able to get using emagged Arcade Battle in one round (40 - 60 minutes )? From my experience you need at least 2 minutes for successful attempt which comes to you every two other attempts where you need to deconstruct machine. So for one maxcap you need 6 minutes (ealier you could get two maxcaps per minute, it was busted). So for 5 bombs you need 30 minutes or more if you get unlucky. I never got more then 6 bombs after the rework and i had to spent entire round to get them (which limits the scope of damage for shuttle bombing in my opinion). Ok, so 30 minutes for 5 bombs. Remember how i told that you need at least Journeyman to even have a chance? You need to farm it which takes another 10 minutes. So we have 40 minutes already, but we are not done yet! Every person that knows about Cuban Pete Arcade will know that you are gaming for maxcaps if you are doing it in the public, so you need to move the console somewhere else, which is even more time and then if you get spotted playing somewhere else its even more obvious (as TTV is a bulky item so you need at least crate to store it somewhere or bluespace bodybag which takes even more time). So lets say we have 40 minutes plus time for preparation, lets say 5 minutes for grabbing tools, deconing console (another place when you can get stopped).

Ok, so we have 45 minutes for bombs alone (i will not include planting or using them, its another thing). How long can you get 50k bomb in sci? People were speedrunning it, good player can do 50k under 20 minutes, not to mention that maxcaps that are meant for blowing stuff up are even faster to make and nobody will question you, you are a scientist making bombs in toxins, who would care? Lets go even further, chem nades, plasmaflooding and other ways that i didnt include are much, much faster, easier, safer for the user and dealier for the victim.

Cuban Pete Arcade Battle is in a spot where its not worth it unless you:
1. Know how to win against him
2. You have no other way of getting the bombs
3. You are ready to spent almost entire round on LRP to get enough bombs for something bigger
4. Have a spot where nobody will notice you and have power (another important part)
And thats why its in a good spot - its a gimmick which rewards committing a lot of time for it (not to mention 4 TC for emag alone) and gives you good reword while putting you at high risk - High Risk, High Reward.

That was long!

TL;DR: PR touches part that is not important and changes nothing.

**Comment by Iamgoofball** [comment]
Back in my day we beat Cuban Pete without any bonus health or damage. I'll remove the bonus health too.

**Comment by Qustinnus** [comment]
back in your day you posted goatse on the github

**Comment by Megarop** [comment]
as if the skill system wasn't already pointless enough

**Comment by CRiPLe1000** [comment]
> Back in my day we beat Cuban Pete without any bonus health or damage. I'll remove the bonus health too.

Back in my day you could get two maxcaps per minute with gaming skill, one maxcap within first 2 minutes with no gaming skill and you couldnt possibly loose. I wrote how much time you need now to even get one maxcap. If you really want to bring back old days you can make Cuban Pete use the old system and then ignore skill (hint - it will be busted again).

Cuban Pete right now without extra health from gaming skill is impossible to win. Its just not, you will be under safe threshold (around 15 health) much, much ealier and gaming skill gives you those extra 10-15 health on Journeyman to get you barely to the end. No sane person would try further if they wouldn't be even close to the end.

**Comment by Winterous** [comment]
I wonder if it would even be mathematically possible to win with the skill health removed... Probably not.

Goof you might be dumb for even considering that.

**Comment by wesoda25** [comment]
All this does is kill something cool that a player can roleplay with and it doesn’t even fix the _actual_ method of cheating, tf?

**Comment by Dorsisdwarf** [comment]
The real nerf to Outbomb Cuban Pete is to make it so once you start playing, deconstructing the arcade will blow you up instantly, thereby removing the ability to negate losses.

**Comment by CRiPLe1000** [comment]
> The real nerf to Outbomb Cuban Pete is to make it so once you start playing, deconstructing the arcade will blow you up instantly, thereby removing the ability to negate losses.

People will find a way around it and you are creating potentially even bigger problem: deconstructing means explosion of the machine -> make Arcade in the public, place C-4/chem nade, snipe it with IED or anything its now a less portable bomb that can be used even ealier. Or gib the user, which is fine i guess. 
But lets perform a thought experiment:
Player spends 15 minutes collecting tools, getting the board, finding good place and farming for the skill needed to even have a chance (something i wrote ealier). They then feel confident enough to emag it and run straight into the situation where they see they cant win it. They know that deconing will gib it so they can either: throw plan into the bin (which is based on random chance anyway) and never try it again or make another Arcade battle with duffel bag full of boards. Cuban Pete Arcade Battle is full of randomness and you try to remove the only part that gives you a chance to get a reward. This pushes Cuban Pete Arcade Battle even more into niche territory, now with added tagline "Why would you even bother with it?". 

I will say it again - its in a good spot, where there are better alternatives but you can still have fun while committing to a gimmick, because this is what it is, a gimmick. Once again, you can make bombs in toxins, atmos and chemistry that are much easier to do and much more certain then this. This PR, if it gets merged in any form, is step one in removing Cuban Pete Arcade Battle, nobody will be using it (because its not fun to commit to a gimmick to be slapped in the face during first try), why keep it in the code.

Lets face it, this PR is not about "removing cheating", its about nerfing it. You can either ignore all of my input or go one of paths listed below :
1. Leave it as it is - its a niche gimmick but it still has some potential plus is a reference to a legacy at this point (and if not, at least you get a cool hat) and knowing when to stop is just raising the skill ceiling (knowledge wise) which is always a good thing.
2. Change the reward for something less powerful while nerfing the Cuban Pete (minor nerf, minor buff) - removing the Charge from emagged version seems good enough in my opinion but it will need some testing as it might be broken again (bonk me on Discord if you would like some further consultation). The reward to keep the theme could be for instance one piece of C-4, Frag Grenade or some small explosive (plus a hat, dont forget about the hat)
3. Remove whole emagged part minus drop of Cuban Pete Hat, no gibbing, no bomb for you, just a hat - you still keep the reference but it will only have use if you want to show that you are a traitor or you stole emag from dead traitor, same as sec mask with Beepsky mode
4. Remove whole emagged part with Cuban Pete Hat, the worst one but at least honest towards playerbase, we removed something, its gone, why bother with the small steps of nerfing it into the ground so its useless and then putting a PR to remove it.

I prefer the option 2, if the original motivation of the PR was a nerf or option 1 if it was random "lore" driven change (he was a bad boi and bad bois don't follow the rules anyway).
I hope you take this long essay into consideration at least.

_why am i doing this again?_

**Comment by LemonInTheDark** [comment]
I don't like this, if the game is currently balanced around high gamer skill (kill me), we shouldn't cut it down to an impossible level. Have I misunderstood something?

File: code/game/machinery/computer/arcade.dm [hunk 1]
```diff
@@ -277,6 +277,8 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 
 	if (!blocked && !gameover)
 		var/attackamt = rand(5,7) + rand(0, gamerSkill)
+		if(obj_flags & EMAGGED) // no cheating for your maxcaps
+			attackamt = rand(5,7)
```
Comment (Rohesie) [inline code/game/machinery/computer/arcade.dm:281]:
```suggestion
		var/attackamt = rand(5,7)
		if(!(obj_flags & EMAGGED)) // no cheating for your maxcaps
			attackamt += rand(0, gamerSkill)
```
This achieves the same you wanted, without rolling the attack twice.

**Comment by Rohesie** [comment]
> 
> 
> I don't like this, if the game is currently balanced around high gamer skill (kill me), we shouldn't cut it down to an impossible level. Have I misunderstood something?

I don't mind the nerf idea (as others have commented, it needs to go further).
The game shouldn't be impossible at unskilled levels, just unlikely to be won.
And cuban pete should definitely be hard to beat, uncheesed by gamer skill grinding.

If you feel strongly about it, feel free to close the PR.
Else, if the requested changes are done, I'm up for merging the nerf.

**Comment by Iamgoofball** [comment]
eh, this is an arcade game, it's a bit silly to nerf it in retrospect, and the thread has given solid arguments as to why this shouldn't be done
